Undefined name 'main' in ./ros2_batch_job/packaging.py

flake8 testing of https://github.com/ros2/ci on Python 3.6.3

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./ros2_batch_job/packaging.py:200:14: F821 undefined name 'main'
    sys.exit(main())
             ^
1     F821 undefined name 'main'
1
```

--- a/ros2_batch_job/packaging.py
+++ b/ros2_batch_job/packaging.py
@@ -194,7 +194,3 @@ def build_and_test_and_package(args, job):
     print('# END SUBSECTION')
 
     return 0
-
-
-if __name__ == '__main__':
-    sys.exit(main())

